### PR TITLE
Feature/eui 51

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-51

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```

[ ] No
```

### CODE REVIEW

File: src/cases/store/reducers/index.ts [hunk 1]
```diff
@@ -2,15 +2,18 @@ import {ActionReducerMap, createFeatureSelector} from '@ngrx/store';
...
   caseCreate: fromCaseCreate.CasesState;
   caseSearchFilter: fromCaseSearchFilters.SearchState;
+  caseListFilter: fromCaseListFilters.CaseFilterState;
 }
 
 export const reducers: ActionReducerMap<State> = {
   caseCreate: fromCaseCreate.reducerCreateCase,
-  caseSearchFilter: fromCaseSearchFilters.reducer
+  caseSearchFilter: fromCaseSearchFilters.reducer,
+  caseListFilter: fromCaseListFilters.reducerCaseListFilter
```
Comment (fazanki) [inline src/cases/store/reducers/index.ts]: can u please rename to caseLIst and please rename caseSearch (i know is not your code)

File: src/cases/store/reducers/index.ts [hunk 2]
```diff
@@ -2,15 +2,18 @@ import {ActionReducerMap, createFeatureSelector} from '@ngrx/store';
 
 import * as fromCaseCreate from './create-case.reducer';
 import * as fromCaseSearchFilters from './search-filter.reducer';
+import * as fromCaseListFilters from './case-list-filter.reducer';
```
Comment (fazanki) [inline src/cases/store/reducers/index.ts]: remove the Filters from both Search and List

File: src/cases/store/reducers/case-list-filter.reducer.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import * as fromCases from '../actions/case-list.filter.action';
```
Comment (fazanki) [inline src/cases/store/reducers/case-list-filter.reducer.ts]: rename file to case-list

File: src/cases/store/reducers/index.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/cases/store/reducers/index.ts]: Done

File: src/cases/store/reducers/case-list-filter.reducer.ts (hunk 3 above)
Comment (udaydenduluri33) [reply src/cases/store/reducers/case-list-filter.reducer.ts]: Done, thanks

File: src/cases/store/reducers/index.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/cases/store/reducers/index.ts]: Done, thanks.

**Comment by udaydenduluri33** [comment]
image

**Comment by udaydenduluri33** [comment]
Screenshot 2019-06-27 at 17 12 26

**Pull request merged**